### PR TITLE
template: update GB mem limit

### PR DESCRIPTION
**Pull request by vrutkovs**
Stage pod get killed with OOMKilled, lets see if we fit into 512MB

### CODE REVIEW

File: dist/openshift/cincinnati.yaml
Comment (LalatenduMohanty) [inline dist/openshift/cincinnati.yaml:232]: I am interested to find out why we need this bump. Is this expected from https://github.com/openshift/cincinnati/pull/174

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: dist/openshift/cincinnati.yaml
Comment (LalatenduMohanty) [reply dist/openshift/cincinnati.yaml:232]: If we change the currency level to 8 can it be done with in 256Mi. @steveeJ ^

Comment (vrutkovs) [reply dist/openshift/cincinnati.yaml:232]: in order to find out what's chewing RAM we need to stop OOMKills first

Comment (LalatenduMohanty) [reply dist/openshift/cincinnati.yaml:232]: https://github.com/openshift/cincinnati/pull/174#discussion_r375927706 so the new cache needs more memory.

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Comment by vrutkovs** [comment]
/hold

Lets see if e2e passes

File: dist/openshift/cincinnati.yaml
Comment (steveej) [reply dist/openshift/cincinnati.yaml:232]: #174 doesn't increase the final cache size, it just fills it up concurrently which uses more memory. I'm curious if it fits into 512MiB as well, let's try it.

**Review by steveej — APPROVED** [review]
/lgtm 
as well

**Comment by LalatenduMohanty** [comment]
> Lets see if e2e passes

Without e2e passing the `lgtm` bot will not merge the PR right?

**Comment by vrutkovs** [comment]
>Without e2e passing the lgtm bot will not merge the PR right?

The bot will merge once all required tests have passed. e2e is not set to required yet

**Comment by steveej** [comment]
> The bot will merge once all required tests have passed. e2e is not set to required yet

It will not merge unless we release the hold though ;-) We are waiting for the e2e tests to pass before we do that, correct?

**Pull request merged**
